com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs

```diff
@@ -94,7 +94,7 @@ public struct iOSNotificationTimeIntervalTrigger : iOSNotificationTrigger
         /// </summary>
         public TimeSpan TimeInterval
```

Comment:
Can you update inline documentation or improve property name, since right now nothing indicates to the user of TimeInterval, that it's in seconds.

Reply:
Since it's a public API, I think it's too minor to change anything more than documentation.